### PR TITLE
Updated questions.md

### PR DESCRIPTION
**Pull request by marwahmanbir**

### CODE REVIEW

**Comment by marwahmanbir** [comment]
Hey @mozzadrella! :smile:

**Comment by marwahmanbir** [comment]
Hey @ericdrosado 👋🏻
I've added you as a collaborator in the module. 😄

**Comment by marwahmanbir** [comment]
Hey @ericdrosado 👋🏻
Just saw your thread, I've added you as a collaborator for this module. smile
I completed the training for Campus Advisors a few weeks ago but didn't get any response since then.

File: questions.md [hunk 1]
```diff
@@ -1 +1,3 @@
 Questions regarding pull requests go here. :)
+
+Describe a 'pull request' in basic terms.
```
Comment (ericdrosado) [inline questions.md]: Hi @marwahmanbir  👋 
Can you `describe a pull request in basic terms`.

Comment (marwahmanbir) [reply questions.md]: Hey, I have updated the file with my answer :)

Comment (marwahmanbir) [reply questions.md]: Yes. I have updated the file with the answer. :)

**Comment by ericdrosado** [comment]
Feel free to continue @marwahmanbir!

**Comment by marwahmanbir** [comment]
Thank you @ericdrosado :smile: 
I have completed all the modules. Are there any further steps in the Campus Advisors training?

**Comment by ericdrosado** [comment]
> I have completed all the modules. Are there any further steps in the Campus Advisors training?

If you would like to be certified through our Campus Advisor program, I would recommend applying here: https://education.github.community/t/completed-the-training-apply-for-certification-here/32724

We are currently working through a backlog of certification requests, so it will take some time. 

We thank you for your patience.

**Comment by marwahmanbir** [comment]
I did apply for the certification as soon as I completed the modules. No issues with the backlog, I'll wait. :slightly_smiling_face:

**Pull request merged**